piper/ratbagd.py

```diff
@@ -158,46 +176,45 @@ def active_profile(self):
             return None
         return self._profiles[self._active_profile]
 
-    @property
-    def capabilities(self):
+    def get_profile_by_index(self, index):
         """
-        Return the capabilities of this device as an array.
-        Capabilities not present on the device are not in the list. Thus use
-        e.g.
-            if RatbagdDevice.CAP_SWITCHABLE_RESOLUTION is in device.caps:
-                 do something
+        Returns the profile found at the given index, or None if no profile was
+        found.
         """
-        return self._caps
+        return self.dbus_call("GetProfileByIndex", "u", index)
 
     def __eq__(self, other):
         return other and self._objpath == other._objpath
 
 class RatbagdProfile(_RatbagdDBus):
     """
-    Represents a ratbagd profile
+    Represents a ratbagd profile.
```

Comment:
fwiw, this is a english vs everything else thing - in english only proper sentences are followed by a full stop. In this case it wouldn't really count, a missing full stop would be appropriate (too). Just FYI